[docs] Unify Tree View `apiRef` methods doc examples (#13193)
mui/mui-x: docs/data/tree-view/rich-tree-view/expansion/expansion.md

{{"demo": "TrackItemExpansionToggle.js"}}

## Imperative API

:::success
To use the `apiRef` object, you need to initialize it using the `useTreeViewApiRef` hook as follows:

```tsx
const apiRef = useTreeViewApiRef();

return <RichTreeView apiRef={apiRef} items={ITEMS}>;
```

> `apiRef` will be undefined during the first render and will then contain methods allowing you to imperatively interact with the Tree View.

Reviewer: ```suggestion
`apiRef` will be `undefined` during the first render. When initialised it will contain methods allowing you to imperatively interact with the Tree View.
```

Author: I think that your version looses the idea that after the 1st render it's good, the "When initialized" does not say that after 1st render it's good

Reviewer: Yeah, makes sense. 
Maybe something like this?

```suggestion
When your component first renders, `apiRef` will be `undefined`. Once the component is fully initialized, `apiRef` holds methods for interacting with the Tree View directly.
```

Reviewer: but feel free to ignore ... 👍🏼

Author: `Once the component is fully initialized` is still something that might be longer than the 1st render IMHO.

How about:

```suggestion
When your component first renders, `apiRef` will be `undefined`. After this initial render,, `apiRef` holds methods to interact imperatively with the Tree View.
```

I also kept "imperatively" because we use it a lot
